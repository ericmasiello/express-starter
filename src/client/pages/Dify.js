/* @flow */
import React, { Component } from 'react';
import {
  Header,
  Navigation,
  Section,
} from 'stencil';
import styles from './Dify.scss';
import Step from '../components/Step';
import * as config from './difyConfig';

export default class Dify extends Component {
  render() {
    return (
      <div>
        <Header
          className={styles.header}
          titleNode="Do It For You Website Design"
          taglineNode="Let us create your website, so you can focus on your business"
          fullHeight
          showPointer
          scrollToElmID={config.LINKS[0].waypoint}
          navigationNode={
            <Navigation
              prioritize
              data-waypoint-nav
              showVistaprintLogo
              links={config.LINKS}
            />}
        />
        <Section
          size="lg"
          className="section--center"
        >
          {config.STEPS.map(step => <Step {...step} key={step.title} />)}
        </Section>
      </div>
    );
  }
}
