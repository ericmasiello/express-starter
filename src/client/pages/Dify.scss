.header {
  background-image: url('../images/seamstress.png');

  :global {
    .header__overlay {
      background-color: rgba(#525557, 0.8);
    }

    .vp-digital-logo {
      margin-bottom: 3.4375rem;
    }
  }
}
