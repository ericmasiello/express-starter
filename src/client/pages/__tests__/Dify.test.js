import React from 'react';
import renderer from 'react-test-renderer';
import Dify from '../Dify';

it('should render', () => {
  const component = renderer.create(
    <Dify />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
