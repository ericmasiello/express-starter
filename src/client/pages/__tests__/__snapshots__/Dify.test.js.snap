// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <header
    className="header header--full-height"
  >
    <div
      className="header__overlay"
    />
    <nav
      className="navigation"
      data-waypoint-nav={true}
    >
      <a
        className="navigation__link navigation__link--on-stuck navigation__link--logo"
        data-waypoint-link={true}
        href="#top"
      >
        <img
          alt="vistaprint digital v"
          className="vp-digital-v-logo"
          height="35"
          src="https://stencil.digital.vistaprint.com/assets/vp-digital-v-logo.svg"
          width="40"
        />
      </a>
      <a
        className="navigation__link"
        data-priority={2}
        data-waypoint-link={true}
        href="#dify-how-it-works"
      >
        How It Works
      </a>
      <a
        className="navigation__link"
        data-priority={4}
        data-waypoint-link={true}
        href="#dify-customer-examples"
      >
        Customer Examples
      </a>
      <a
        className="navigation__link"
        data-priority={3}
        data-waypoint-link={true}
        href="#dify-features"
      >
        Features
      </a>
      <a
        className="navigation__link"
        data-priority={1}
        data-waypoint-link={true}
        href="#dify-pricing"
      >
        Pricing
      </a>
    </nav>
    <div
      className="header__container"
    >
      <div
        className="header__content"
      >
        <img
          alt="vistaprint digital"
          className="vp-digital-logo"
          height="60"
          src="https://stencil.digital.vistaprint.com/assets/vp-digital-logo.svg"
          width="120"
        />
        <h1
          className="t-jumbo-fluid u-text-shadow"
        >
          Do It For You Website Design
        </h1>
        <h2
          className="t2-fluid u-text-shadow"
        >
          Let us create your website, so you can focus on your business
        </h2>
      </div>
      <a
        className="header__pointer"
        data-waypoint-link-first={true}
        href="#dify-how-it-works"
      >
        <svg
          className="icon icon--md icon-chevron-down header__pointer__icon"
          height={32}
          viewBox="0 0 64 64"
          width={32}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M55.4 21.3L33.6 45.1c-.1.1 0 .2-.1.3-.5.4-1 .6-1.5.6s-1-.2-1.4-.5c-.1-.1-.1-.2-.1-.3L8.6 21.3c-.7-.7-.7-2 0-2.7.7-.7 1.9-.7 2.7 0L32 41.2l20.7-22.6c.7-.7 1.9-.7 2.7 0 .8.7.8 1.9 0 2.7z"
          />
        </svg>
      </a>
    </div>
  </header>
  <section
    className="section section--lg section--center"
  >
    <div
      className={undefined}
    >
      <div
        className="card__number"
      >
        1
      </div>
      <h1
        className="t3"
      >
        Submit Design Brief
      </h1>
      <p
        className="t5"
      >
        Fill out a creative brief to explain your business and your vision for your site.
      </p>
    </div>
    <div
      className={undefined}
    >
      <div
        className="card__number"
      >
        2
      </div>
      <h1
        className="t3"
      >
        Schedule Phone Consultation
      </h1>
      <p
        className="t5"
      >
        Fill out a creative brief to explain your business and your vision for your site.
      </p>
    </div>
    <div
      className={undefined}
    >
      <div
        className="card__number"
      >
        3
      </div>
      <h1
        className="t3"
      >
        Review Website
      </h1>
      <p
        className="t5"
      >
        Give feedback and collaborate on changes to create the best design for you.
      </p>
    </div>
    <div
      className={undefined}
    >
      <div
        className="card__number"
      >
        <svg
          className="icon icon--xs icon-tick"
          height={16}
          viewBox="0 0 64 64"
          width={16}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M63.4 13.4l-40 40c-.8.8-2 .8-2.8 0h-.1L.7 33.5c-.8-.8-.8-2 0-2.8.8-.8 2-.8 2.8 0L22 49.2l38.6-38.6c.8-.8 2-.8 2.8 0 .7.8.7 2 0 2.8z"
          />
        </svg>
      </div>
      <h1
        className="t3"
      >
        Accept Design
      </h1>
      <p
        className="t5"
      >
        Accept the design for your site and go live. You can continue to work with us to make changes going forward, or edit yourself in our website builder tool.
      </p>
    </div>
  </section>
</div>
`;
